map/bookmark_manager.hpp

```diff
@@ -760,6 +764,8 @@ class BookmarkManager final
   kml::TrackId m_selectedTrackId = kml::kInvalidTrackId;
   m2::PointF m_maxBookmarkSymbolSize;
 
+  std::unique_ptr<Bookmark> m_recentlyDeletedBookmark = nullptr;
```

Comment:
nit: There is no need to explicitly set a smart pointer to nullptr. It is empty/uninitialized/nullptr by default.
```suggestion
  std::unique_ptr<Bookmark> m_recentlyDeletedBookmark;
```